### ISSUE TITLE
Importing @aws-amplify/ui-react/styles.css will override global font-family

### ISSUE THREAD

**Issue by lacion** [aws-amplify/amplify-ui#1062]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

React

### Which UI component?

Authenticator

### How is your app built?

Create React App

### Please describe your bug.

Importing `@aws-amplify/ui-react/styles.css` needs to be done before any other custom CSS otherwise html -> font-family will be overridden and custom fonts defined in the Theme will never take effect

### What's the expected behaviour?

Creating a custom theme with a font definition should change the UI font and not conflict with my own app CSS

```javascript
const myTheme = {
  name: 'my-theme',
  tokens: {
    fonts: {
      default: {
        variable: { value: 'Raleway' },
        static: { value: 'Raleway' },
      },
    },
  },
};
```

### Help us reproduce the bug!

Simply import `@aws-amplify/ui-react/styles.css` after any other CSS and Amplify UI CSS html {font-family} CSS definition will be the one use globally

### Code Snippet

```javascript
const myTheme = {
  name: 'my-theme',
  tokens: {
    fonts: {
      default: {
        variable: { value: 'Raleway' },
        static: { value: 'Raleway' },
      },
    },
  },
};
```
